packages/hash/api/src/model/file.model.ts

```diff
@@ -1,10 +1,11 @@
+/** @todo - Fix this file */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
(Comment not for line)
Could you remove the file export from the `src/model/index.ts` file at the bottom? It's causing import errors as it extends `Entity` which has been removed

Reply:
I'm not seeing any errors on my end, is this just in VSCode?

Reply:
It's when running the backend

Reply:
So it is, thanks! Have pushed the fix mentioned